platform/security/certificate/security-ocsp-checker/src/main/java/org/codice/ddf/security/ocsp/checker/OcspChecker.java

```diff
@@ -394,14 +408,28 @@ private CertificateStatus getStatusFromOcspResponse(OCSPResp ocspResponse) {
       if (singleResps == null) {
         return new UnknownStatus();
       }
-
-      SingleResp response = Arrays.stream(singleResps).findFirst().orElse(null);
+      SingleResp response =
+          Arrays.stream(singleResps)
+              .filter(singleResp -> singleResp.getCertID() != null)
+              .collect(Collectors.toList())
```

Comment:
You don't need to collect in the middle of the stream, you can just go straight from your first filter to your second filter.

Reply:
Fixed.